apps/mobile/src/screens/NftDetailScreen/DetailSection.tsx

```diff
@@ -19,7 +20,7 @@ export function DetailSection({
 export function DetailLabelText({ children }: PropsWithChildren) {
   return (
     <Typography
-      className="text-xs text-metal dark:color-metal"
+      className="text-xs text-[#707070] dark:color-metal"
```

Comment:
@Robinnnnn need to find out how to not hardcore color here

Reply:
i would try `shadow` or `faint`

<img width="313" alt="image" src="https://github.com/gallery-so/gallery/assets/12162433/f52f81b9-056d-4300-9d77-d8d1e7f8fb2d">

Reply:
i think you want shadow since this is in light mode